addons/acodec/voc.c

```diff
@@ -0,0 +1,342 @@
+/*
+ * Allegro5 Creative Voice Audio Reader.
```

Comment:
I'll add it in a separate commit but seems that all around the Addon section of code the "Allegro" ASCII art is mostly (if not ubiquitously) absent. ^__^", that's why I havent put it there.